TemporaryProjects/TemporaryProjects.csproj

```diff
@@ -35,6 +32,7 @@
     <StartAction>Program</StartAction>
     <StartProgram Condition="'$(DevEnvDir)' != ''">$(DevEnvDir)devenv.exe</StartProgram>
     <StartArguments>/rootsuffix Exp</StartArguments>
+    <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
```

Comment:
This is not strictly necessary when this project already contains PackageReference - VS will always keep using the current format (packages.config vs PackageReference). This only ensures that if all packages were removed, VS would choose PackageReference as the default.

Reply:
Thanks for the tip.